app/views/gobierto_admin/gobierto_plans/projects/index.html.erb

```diff
@@ -42,7 +42,7 @@
         <% end %>
       </td>
       <td><%= link_to_if current_admin_actions.include?(:edit), project.name, edit_admin_plans_plan_project_path(@plan, project), data: { turbolinks: false } %></td>
-      <td><%= format_percentage(project.progress) %></td>
+      <td><%= project.progress %></td>
```

Comment:
Why did you remove this helper? Can you restore it?

Reply:
into my local test raise undefined method. but let's try with CI